### PR TITLE
Write float values using the invariant culture.

### PR DESCRIPTION
**Pull request by j0rn**
### Prerequisites

- [X] I have written a descriptive pull-request title
- [X] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [X] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [X] I have provided test coverage for my change (where applicable)

### Description
Float values a written using the current users culture.  Tests fails for "non-english" culture.

    StringRepresentationsAreCorrect(color: CieLab(42,4, 94,5, 83,4), text: "CieLab(42.4, 94.5, 83.4)") [FAIL]
      Assert.Equal() Failure
                          (pos 9)
      Expected: CieLab(42.4, 94.5, 83.4)
      Actual:   CieLab(42,4, 94,5, 83,4)
                          (pos 9)

### CODE REVIEW

File: src/ImageSharp/ColorSpaces/CieLab.cs [hunk 1]
```diff
@@ -127,7 +128,7 @@ public override int GetHashCode()
         }
 
         /// <inheritdoc/>
-        public override string ToString() => $"CieLab({this.L:#0.##}, {this.A:#0.##}, {this.B:#0.##})";
+        public override string ToString() => $"CieLab({FloatToString(this.L, this.A, this.B)})";
```
Comment (tocsoft) [inline src/ImageSharp/ColorSpaces/CieLab.cs]: inset of adding `FloatToStringUtil` (and the extra allocation that involves) you should use this syntax instead:

```
public override string ToString() => FormattableString.Invariant($"CieLab({this.L:#0.##}, {this.A:#0.##}, {this.B:#0.##})");
```

Comment (j0rn) [reply src/ImageSharp/ColorSpaces/CieLab.cs]: image

ColorSpaces\CieXyz.cs(97,46): error CS0103: The name 'FormattableString' does not exist in the current context [ImageSharp.csproj]

Comment (dlemstra) [reply src/ImageSharp/ColorSpaces/CieLab.cs]: We probably should

Comment (JimBobSquarePants) [reply src/ImageSharp/ColorSpaces/CieLab.cs]: Yup Windows phone is dead now and the only reason I clung on.

https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support

Comment (iamcarbon) [reply src/ImageSharp/ColorSpaces/CieLab.cs]: I submitted PRs across the repos to increase the NETSTANDARD target to v1.3.

Comment (JimBobSquarePants) [reply src/ImageSharp/ColorSpaces/CieLab.cs]: Merged, and I've updated this PR from master so should be good to use now.

Comment (j0rn) [reply src/ImageSharp/ColorSpaces/CieLab.cs]: Comitted with FormattableString.Invariant.

**Comment by tocsoft** [comment]
@j0rn thanks for the fix 👍

**Pull request merged**